File: client/iOS/SwaggerClientApp/SwaggerClientApp/ViewController.m
//
//  ViewController.m
//  SwaggerClientApp
//
//  Created by Loreto Parisi on 19/09/16.
//  Copyright © 2016 Musixmatch. All rights reserved.
//

#import "ViewController.h"
#import "SwaggerClient.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    
    SWGConfiguration *apiConfig = [SWGConfiguration sharedConfig];
    
    // Configure API key authorization: (authentication scheme: key)
    [apiConfig setApiKey:@"YOUR_API_KEY" forApiKeyIdentifier:@"apikey"];
    // Uncomment below to setup prefix (e.g. Bearer) for API key, if needed
    //[apiConfig setApiKeyPrefix:@"Bearer" forApiKeyIdentifier:@"apikey"];
    
    
    NSString *albumId = @"14250417"; // The musiXmatch album id
    NSString *format = @"json"; // output format: json, jsonp, xml. (optional) (default to json)
    
    SWGAlbumApi *apiInstance = [[SWGAlbumApi alloc] init];
    
    //
    [apiInstance albumGetGetWithAlbumId:albumId
                                 format:format
                               callback:nil
                      completionHandler: ^(SWGInlineResponse200* output, NSError* error) {
                          if (output) {
                              NSLog(@"%@", output);
                          }
                          if (error) {
                              NSLog(@"Error: %@", error);
                          }
                      }];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: client/xamarin/SwaggerClient/iOS/ViewController.cs
using System;

using UIKit;

using Newtonsoft.Json;
using IO.Swagger.Api;
using IO.Swagger.Client;
using IO.Swagger.Model;

namespace SwaggerClient.iOS
{
	public partial class ViewController : UIViewController
	{
		int count = 1;

		public ViewController(IntPtr handle) : base(handle)
		{
		}

		public override void ViewDidLoad()
		{
			base.ViewDidLoad();

			// Code to start the Xamarin Test Cloud Agent
#if ENABLE_TEST_CLOUD
			Xamarin.Calabash.Start ();
#endif

			// Perform any additional setup after loading the view, typically from a nib.
			Button.AccessibilityIdentifier = "myButton";
			Button.TouchUpInside += delegate
			{
				var title = string.Format("{0} clicks!", count++);
				Button.SetTitle(title, UIControlState.Normal);

				// Configure API key authorization: key
				Configuration.Default.ApiKey.Add("apikey", "YOUR_API_KEY");
				// Uncomment below to setup prefix (e.g. Bearer) for API key, if needed
				// Configuration.Default.ApiKeyPrefix.Add("apikey", "Bearer");

				var apiInstance = new AlbumApi();
				var albumId = "14250417";   // string | The musiXmatch album id
				var format = "json";  // string | output format: json, jsonp, xml. (optional)  (default to json)

				try
				{
					// 
					InlineResponse200 result = apiInstance.AlbumGetGet(albumId, format);
					var res = JsonConvert.SerializeObject(result);
					Console.WriteLine(res);
				}
				catch (IO.Swagger.Client.ApiException e)
				{
					Console.WriteLine("Exception when calling DefaultApi.AlbumGetGet: " + e.Message);
				}
				catch (Exception e)
				{
					Console.WriteLine("Exception when calling DefaultApi.AlbumGetGet: " + e.Message);
				}
			};
		}

		public override void DidReceiveMemoryWarning()
		{
			base.DidReceiveMemoryWarning();
			// Release any cached data, images, etc that aren't in use.		
		}
	}
}
